ui/pages/providers/add.js

```diff
@@ -302,6 +435,20 @@ export default function ProvidersAddDetails() {
                   </p>
                 )}
               </div>
+              <div>
+                <input
+                  id='scim-checkbox'
+                  type='checkbox'
+                  value=''
+                  class='h-4 w-4 rounded border-gray-300 bg-gray-100 text-blue-600 focus:ring-2 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:ring-offset-gray-800 dark:focus:ring-blue-600'
```

Comment:
why do we need to styling the dark mode?

Reply:
Good eye. This was from the component I was basing it off, missed it in this case. Removed it.